augur/cli/configure.py

```diff
@@ -164,6 +164,11 @@
                 "switch": 1,
                 "workers": 1
             },
+            "release_worker": {
+                "port": 52000,
```

Comment:
Can you set this default port to be `51100`? We try to leave 100 ports open after each worker and we try to keep them close together if we can, it's a bit easier to keep track of that way IMO.

Reply:
Thanks for noticing @ccarterlandis